dist/index.html

```diff
@@ -9,10 +9,26 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap">
     <link rel="stylesheet" href="./index.css">
-<script defer src="bundle023585e19f2dc506e145.js"></script></head>
-<body>
-    <h1>TV Shows </h1> 
+<script defer src="bundle3b1ce28a982240b850b0.js"></script></head>
+<body class="container">
+    <header>
+        <div class="logo">Logo</div>
+        <nav>
```

Comment:
Great works, 
I have only one suggestion which I think should make the navigation bar looks better.

Currently the nav items are too close and in center. If we spread these using space-between, it will looks even better like the follow:
![localhost_3000_](https://user-images.githubusercontent.com/49184579/163991063-90074ab8-1c7c-4064-94ec-1282a387d3c3.png)